### PR TITLE
can I get some eyes on this blog?

### PR DESCRIPTION
**Pull request by tnh**
can I get some eyes on this blog?

I must warn ya'll - the grammar is correct in my head, but sometimes the grammar in my fingers typing the thing might not be :)

### CODE REVIEW

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+author: Trent Hornibrook
+images: blog/failure.jpg
+
+
+excerpt:
+    As a CTO, how would you react when a mistake was made that caused 80 developers to stop working for two days?
+
+---
+
+I want to share a story about a major development outage that occurred at my former workspace in 2012 and the corresponding leadership response.
```
Comment (jags14385) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: could reword to "I want to share my experience when a major development outage occurred and how the leadership responded then" 

No specifics such as 2012  & so .

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+My former workplace began aggressively using AWS. Like most workplaces that are familiar with a datacenter operating model, cost accounting becomes a focus given the pay-per-hour operating expenditure pricing model of AWS. Generally, the first solution of arresting cost is to implement a 'stopinator' type tool that will switch off all compute resources out-of-business hours.
```
Comment (jags14385) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: We were aggressively using AWS.

What is "stopinator" ?? more context may be ..

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+My former workplace at the time invested heavily on individual local environments deployed into AWS along with team specific CI's also deployed into AWS. There were some 20 different complex environments supporting 80 odd IT engineers.
```
Comment (jags14385) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: We invested..

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+A developer implemented a custom 'stopinator' however they introduced two fundamental bugs that caused a critical outage.
```
Comment (jags14385) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: caused a critical outage : 

 - reason 1
- reason 2

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+---
+
+ I want to share my experience when a major development outage occurred and how the leadership responded.
+
+We were aggressively using AWS. Like most workplaces that are familiar with a datacenter operating model, cost accounting becomes a focus given the pay-per-hour operating expenditure pricing model of AWS. Generally, the first solution of arresting cost is to implement a solution that would turn off EC2 instances out of hours. This was our in-house 'stopinator' tool.
+
+We invested heavily on individual local environments deployed into AWS along with team specific CI's also deployed into AWS. There were some 20 different complex environments supporting 80 odd IT engineers.
+
+A developer implemented a custom 'stopinator' however they introduced two fundamental bugs that caused a critical outage:
```
Comment (jags14385) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: 'stopinator' solution ,

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+You can imagine the screams of "what happened to my environment?" or "where did my Jenkins go?". Sure, cost savings was in full swing, however some 80 people were fully impacted by this destructive event. The damage was massive - particularly for the Continuous Integration servers, as the pipeline build logic was manually created and was subsequently lost through the event.
+
+The outage lasted multiple days as teams rewrote their CI logic. I was one of the engineers who rebuild CIs whilst being completely annoyed and angry at the event that caused this outage.
+
+The CTO responded to the event in a way that left me dismayed - he celebrated the person that made the mistake. I remember being completely bewildered of the response though now reflecting back, I think it was a great response.
+
+The problem was not that an engineer made a mistake. The problem was that the systems were built in a way that were not resilient to failure.
+
+When you build anything, design it for failure - as failure will occur.
```
Comment (jags14385) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: as failures will occur.

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+We invested heavily on individual local environments deployed into AWS along with team specific CI's also deployed into AWS. There were some 20 different complex environments supporting 80 odd IT engineers.
+
+A developer implemented a custom 'stopinator' however they introduced two fundamental bugs that caused a critical outage:
+
+* The time-period logic was incorrect - meaning that it ran during business hours
+* The dry-run logic was incorrect - meaning when it ran, it destroyed everything
+
+You can imagine the screams of "what happened to my environment?" or "where did my Jenkins go?". Sure, cost savings was in full swing, however some 80 people were fully impacted by this destructive event. The damage was massive - particularly for the Continuous Integration servers, as the pipeline build logic was manually created and was subsequently lost through the event.
+
+The outage lasted multiple days as teams rewrote their CI logic. I was one of the engineers who rebuild CIs whilst being completely annoyed and angry at the event that caused this outage.
```
Comment (jags14385) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: The transition from the introduction to the matter , is very sudden. Can we reword it to smooth it out.

**Comment by mngau** [comment]
Is there a better quality image it's really grainy and blurry.

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+excerpt:
+    As a CTO, how would you react when a mistake was made that caused 80 developers to stop working for two days?
+
+---
+
+ I want to share my experience when a major development outage occurred and how the leadership responded.
+
+We were aggressively using AWS. Like most workplaces that are familiar with a datacenter operating model, cost accounting becomes a focus given the pay-per-hour operating expenditure pricing model of AWS. Generally, the first solution of arresting cost is to implement a solution that would turn off EC2 instances out of hours. This was our in-house 'stopinator' tool.
+
+We invested heavily on individual local environments deployed into AWS along with team specific CI's also deployed into AWS. There were some 20 different complex environments supporting 80 odd IT engineers.
```
Comment (mngau) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: We invested heavily on individual local environments along with team specific CI's that were both deployed into AWS. This made up around 20 different complex environments which supported 80 odd IT engineers.

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+A developer implemented a custom 'stopinator' however they introduced two fundamental bugs that caused a critical outage:
+
+* The time-period logic was incorrect - meaning that it ran during business hours
+* The dry-run logic was incorrect - meaning when it ran, it destroyed everything
+
+You can imagine the screams of "what happened to my environment?" or "where did my Jenkins go?". Sure, cost savings was in full swing, however some 80 people were fully impacted by this destructive event. The damage was massive - particularly for the Continuous Integration servers, as the pipeline build logic was manually created and was subsequently lost through the event.
```
Comment (mngau) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: Just imagine the screams of "What happened to my environment?" or "Where did my Jenkins go?". Sure, minimising our AWS costs was in full swing, however at least 80 people were entirely impacted by this very destructive event. The resulting blast radius of damage was especially massive - the Continuous Integration  (CI) servers in particular, due to the fact that pipeline build logic had been manually created and was subsequently lost following the event.

**Comment by tnh** [comment]
@mngau there is - i resized the image down hoping it would render at the image size - but its not. I'll put the full size back.

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown (hunk 6 above)
Comment (mngau) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: When you build anything, design it to be resilient to failure; as failure will always occur.

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown (hunk 7 above)
Comment (mngau) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: I was one of the engineers who had the task of rebuilding CIs, all the while feeling completely annoyed and angry about the event that caused this outage and bulk of unplanned work.

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+The outage lasted multiple days as teams rewrote their CI logic. I was one of the engineers who rebuild CIs whilst being completely annoyed and angry at the event that caused this outage.
+
+The CTO responded to the event in a way that left me dismayed - he celebrated the person that made the mistake. I remember being completely bewildered of the response though now reflecting back, I think it was a great response.
```
Comment (mngau) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: I remember being completely bewildered by his response, although upon reflection as I now think back, it was a great response.

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+
+The problem was not that an engineer made a mistake. The problem was that the systems were built in a way that were not resilient to failure.
```
Comment (mngau) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: Our systems were built in a way that meant they were not resilient to failures, that was the real problem.

**Comment by tnh** [comment]
@jags14385  / @mngau Thoughts? 👯‍♂️  👯

**Comment by mngau** [comment]
Yup just waiting for the build to finish so I can check it on the beta.

File: _posts/2017-08-30-the-outage-that-stopped-all-development.markdown [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+author: Trent Hornibrook
+images: blog/failure.jpg
+
+
+excerpt:
+    As a CTO, how would you react when a mistake was made that caused 80 developers to stop working for two days?
+
+---
+
+ I want to share my experience when a major development outage occurred and how the leadership responded.
```
Comment (mngau) [inline _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: I want to share my experience of when a major development outage occurred, and how the leadership responded.

Comment (mngau) [reply _posts/2017-08-30-the-outage-that-stopped-all-development.markdown]: Maybe a slight re-word to the start. I only just noticed it.

**Comment by tnh** [comment]
This is the 🥇  @mngau
